test/basic.test.js

```diff
@@ -344,3 +344,26 @@ test('render a single quote as JSON', (t) => {
   t.equal(output, JSON.stringify(toStringify))
   t.ok(validate(JSON.parse(output)), 'valid schema')
 })
+
+test('should not break when the value of field is null', (t) => {
```

Comment:
we should add more tests:

The test without the type's array
```
{
    type: 'object',
    properties: {
      updatedAt: {
        type: 'string'
      }
    }
  }
```

The test with nullable
```
{
    type: 'object',
    properties: {
      updatedAt: {
        type: ['string', 'null']
      }
    }
  }
```

Reply:
sorry ignore this I didn't read the comment below